swagger.yaml

```diff
@@ -580,6 +580,64 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+                
+  "/severa/users/{severaUserId}/workdays":
+    get:
+      operationId: getWorkdaysForUser
```

Comment:
Rather than getWorkdaysForUser or Get user's workdays for a specific time period in the spec lets be explicit if we are finding or listing, so in this case listWorkdaysForUser and List user's workdays....

That should match the majority of the already exisitng endpoint in the spec hopefully